internal/cli/atlas/atlas.go

```diff
@@ -86,7 +86,7 @@ func Builder() *cobra.Command {
 		events.Builder(),
 		metrics.Builder(),
 		performanceadvisor.Builder(),
-		logs.Builder(),
+		logs.MongoCLIBuilder(),
```

Comment:
👀  mongocli in atlas? also the commend on the method says "until atlas is released" but atlas was already released

Reply:
I made the comment I think it should be "Until atlas commands are deprecated in MongoCLI". 

AFAIK this builder is used only by MongoCLI, hence the naming, fine w/ any naming though 👍